src/components/BalanceHistory.js

```diff
@@ -2,22 +2,21 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Chart from 'react-apexcharts';
 import Typography from '@material-ui/core/Typography';
+import moment from 'moment';
 
 const BalanceHistory = ({ initialTransactions }) => {
   let result = [];
 
   initialTransactions.reduce((previous, current) => {
-    const value = previous - parseInt(current.amount, 10);
-    result = result.concat(value);
-    return value;
+    const balance = previous + parseInt(current.amount, 10);
```

Comment:
I think it might be better to allow decimals here and round up the final balance towards the end. You can do something like this `+current.amount` or `Number(current.amount)` both would do the job!

Reply:
Yes, don't know why I did not even think about that I was parsing it to an int... However the Number(current.amount) did not parse it right so I just left it as it is. I need to change it later when I have time